### ISSUE TITLE
Shared Dimension not working

### ISSUE THREAD

**Issue by CDiGallo** [zazuko/cube-creator#1442]
**Describe the bug**
I have the BFS Identifiers. They are recognized by CC. (The municipalities seem to be too much and I get an 502 network error, but this is probably another issue).
After I transform my cube, it still has only the identifiers in the dimension. Visualize also does not recognize the dimension as a shared dimension.

**Affected functionalities** (all that apply)

* [x ] Transformation
* [ x] Other

**Relevant links**
 Here is the link for the problems with the [cantons](https://int.cube-creator.lindas.admin.ch/app/cube-projects/cube-project!!zweitwohnungen-kantone-pj6odxb05mo/metadata)

Here is the link for the municipalities, but it may be another issue: https://int.cube-creator.lindas.admin.ch/app/cube-projects/cube-project!!zweitwohnungen-gemeinden-0ypdk6djzxo/materialize

**To Reproduce**
1. Go to 'Cube Designer'
2. Click on 'dimension'
3. it recognizes the identifier
4. CC does not make the link.

**Expected behavior**
The identifier makes a link  the shared dimension. I can click on the cell and get something akin to https://ld.admin.ch/canton/11

For the municipalities, after I use the "autofill" and it recognizes (most) of the municipalities and I click "update mappings" I expect not "Error: Unknown error".

**Desktop (please complete the following information):**
 - OS: [W10]
 - Browser [edge]

**Comment by CDiGallo** [comment]
@Rdataflow Yes I tried. Did not work Thanks anyway.

Update:
Today it just worked with the cantons (I did not even click on anything. I opened CC and it just was as it needed to be).

But the municipalities still refuse to link. After clicking "update mapping"  and wating for >60 sec I get an error in CC: "Access denied".

In the browser developer-tools I get the following error:
/cube-project/zweitwohnungengemeinden-gxj8bozzdkx/dimension-mapping/gdenr-ggemda5ogmj:1 
        
        
        Failed to load resource: the server responded with a status of 401 (Unauthorized)
instrument.js:109  Error: Access denied
    at a.fromResponse (errors.ts:54:16)
    at async Object.invokeSaveOperation (index.ts:153:7)
    at async l (use-hydra-form.ts:57:13)


It also sends me to the following link: https://[int.cube-creator.lindas.admin.ch/cube-project/zweitwohnungengemeinden-gxj8bozzdkx/dimension-mapping/gdenr-ggemda5ogmj](https://int.cube-creator.lindas.admin.ch/cube-project/zweitwohnungengemeinden-gxj8bozzdkx/dimension-mapping/gdenr-ggemda5ogmj)

@tpluscode
